app/MetricUnitConverter.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace ConverterApp;
+
+class MetricUnitConverter extends Converter
+{
+public function converter($currentUnit,$newUnit)
```

Comment:
You are accepting value not unit. so name of input param should be according to that.